### PR TITLE
CNS Guidelines for OCP 3.5

### PR DESCRIPTION
**Pull request by ahardin-rh**
Continues work from https://github.com/openshift/openshift-docs/pull/4103

### CODE REVIEW

File: install_config/persistent_storage/persistent_storage_glusterfs.adoc [hunk 1]
```diff
@@ -38,6 +38,76 @@ separately in the following Red Hat Gluster Storage documentation:
...
+When deploying applications on {product-title}, it is important to estimate how
+long it will take to build an environment, in regard to storage, so that an
+application pod is able to start.
+
+If the application pod requires a persistent volume claim (PVC), then you must
+consider the time it takes for a different number of CNS to be created and bound
+to the corresponding PVC.
+
+Creation time of CNS volumes scales linearly up to 100 volumes. In the latest
+tests, each volume took approximately six seconds to be created, allocated, and
```
Comment (adellape) [inline install_config/persistent_storage/persistent_storage_glusterfs.adoc]: s/six/6/

Since 100 was used in the same para.

**Review by adellape — COMMENTED** [review]
:+1:

File: install_config/persistent_storage/persistent_storage_glusterfs.adoc [hunk 2]
```diff
@@ -38,6 +38,76 @@ separately in the following Red Hat Gluster Storage documentation:
...
+[[volumes-creation-time-for-container-native-storage]]
+==== Creation Time of Volumes with Container Native Storage
+
+When deploying applications on {product-title}, it is important to estimate how
+long it will take to build an environment, in regard to storage, so that an
+application pod is able to start.
+
+If the application pod requires a persistent volume claim (PVC), then you must
+consider the time it takes for a different number of CNS to be created and bound
+to the corresponding PVC.
```
Comment (bfallonf) [inline install_config/persistent_storage/persistent_storage_glusterfs.adoc]: I'm a little confused about the above two paras. Can I suggest something like: 

"Building environment storage can influence the time it takes for an application to start. For example, if the application pod requires a persistent volume claim (PVC), then extra time might have to be considered for CNS to be created and bound to the corresponding PVC. This effects the build time for an application pod to start."

File: install_config/persistent_storage/persistent_storage_glusterfs.adoc [hunk 3]
```diff
@@ -38,6 +38,76 @@ separately in the following Red Hat Gluster Storage documentation:
...
+Dynamic storage provisioning and storage classes were also configured and used
+when provisioning the PVC.
+
+[[volumes-deletion-time-for-container-native-storage]]
+==== Deletion Time of Volumes with Container Native Storage
+
+When you delete a PVC that is used by an application pod, then that action
+will trigger the deletion of the CNS volume that was used by the PVC.
+
+PVCs will disappear immediately from `oc get pvc` view. However, the the time to
```
Comment (bfallonf) [inline install_config/persistent_storage/persistent_storage_glusterfs.adoc]: s/from/from the

maybe s/view/output ?

Comment (bfallonf) [inline install_config/persistent_storage/persistent_storage_glusterfs.adoc]: Also, double "the"

**Comment by ahardin-rh** [comment]
@bfallonf @adellape Thanks for the quick feedback!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/persistent_storage/persistent_storage_glusterfs.adoc#install-config-persistent-storage-persistent-storage-glusterfs[Configuring Persistent Storage -> Persistent Storage Using GlusterFS]
|Added xref:../install_config/persistent_storage/persistent_storage_glusterfs.adoc#container-native-storage-recommendations[Container Native Storage Recommendations].
%

**Pull request merged**